frontend/Peerprep/README.md

```diff
@@ -12,9 +12,10 @@ npm install
 ```
 
 ## Env Variables
+
 - Set up environment varaibles in a `.env` file
   - `FIREBASE_API_KEY` which you can get from the firebase console.
-
+  - `HOST_ADDRESS` = localhost or the static ip.
```

Comment:
Does this mean that the microservices are exposed to the Internet?

Reply:
yea, shld be able to be access with port number

Reply:
Is there a way to keep it hidden from the internet, so that only the microservices can access them? Because the backend APIs are not supposed to be internet facing.

Reply:
use server actions for frontend to call backend services except for websocket. therefore, no need to expose the backend apis